packages/react-components/src/components/RemoteVideoTile.tsx

```diff
@@ -54,6 +55,7 @@ export const _RemoteVideoTile = React.memo(
     drawerMenuHostId?: string;
     onPinParticipant?: (userId: string) => void;
     onUnpinParticipant?: (userId: string) => void;
+    updateScalingModeForRemoteUser?: (userId: string, scalingMode: ViewScalingMode) => void;
```

Comment:
Is this a public api change? I guess not

Reply:
No, just added it to remote video tile which is internal component